src/main/typescript/components/overview/DepositTableRow.tsx

```diff
@@ -32,46 +27,46 @@ const Enterable = ({ title }: EnterableProps) => (
 )
 
 interface DepositTableRowProps {
-    depositId: DepositId
     deposit: Deposit
     deleting?: DeleteState
-
     deleteDeposit: () => void
+    editable: Boolean
+    enterDeposit: () =>  void
 }
 
-const DepositTableRow = ({ depositId, deposit, deleting, deleteDeposit }: DepositTableRowProps) => {
-    const editable = isEditable(deposit)
+class DepositTableRow extends Component<DepositTableRowProps> {
```

Comment:
now that you have this as just a `Component` with a `render` again, you can revert it back to a functional component.